lib/toolbar/edit-toolbar.vue

```diff
@@ -337,9 +343,8 @@
     }),
     watch: {
       snackbar(val) {
-        if (val) {
+        if (Object.keys(val).length > 0) {
```

Comment:
We could have the condition without evaluating `> 0`
```suggestion
        if (Object.keys(val).length) {
```

Reply:
👍 changed